transparentdemocracy/actors/actors.py

```diff
@@ -1,130 +1,91 @@
 import asyncio
-import json
-import os
+import logging
+import ssl
+from pathlib import Path
 
 import aiofiles
 import aiohttp
+from aiohttp import ClientSession
 
 from transparentdemocracy import CONFIG
 
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
 
 class ActorHttpGateway:
     def __init__(self, config, base_url='https://data.dekamer.be/v0/actr'):
         self.config = config
         self.base_url = base_url
+        self.actors_path = Path(config.actor_json_input_path())
+        self.actors_path.mkdir(parents=True, exist_ok=True)
 
-    async def download_pages(self, max_pages, max_concurrent_requests=5):
-        os.makedirs(self.config.actor_json_pages_input_path(), exist_ok=True)
-        queue = asyncio.Queue(min(max_concurrent_requests, max_concurrent_requests))
-        stop_event = asyncio.Event()
+    async def download_actors(self, max_pages: int, max_concurrent_requests: int = 5):
+        ssl_context = ssl.create_default_context()
+        ssl_context.check_hostname = False
```

Comment:
@jellevictoor Should not be necessary with the right certificate setup. I'm not sure on the exact procedure, but google "python ssl certificate error" for more info.